mfr/core/remote_logging.py

```diff
@@ -22,8 +19,7 @@
 
     keen_payload = copy.deepcopy(metrics)
     keen_payload['meta'] = {
-        'mfr_version': mfr.__version__,
-        # 'wb_version': waterbutler.__version__,
```

Comment:
I remove this because I am not sure how this would work in the new code. #SideEffect